src/components/ScaleBar.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from "react";
+
+import styles from "./scalebar.module.css";
+import { MICRO } from "../constants";
+import { SimulariumContext } from "../simulation/context";
+import { AGENT_AB_COLOR } from "../constants/colors";
+
+const ScaleBar: React.FC = () => {
+    const { maxConcentration } = useContext(SimulariumContext);
+    const labelArray = [];
+    for (let i = maxConcentration; i >= 0; i = i - 2) {
```

Comment:
Is there an assumption that maxConcentration is integer and even?  Do you allow the maxConcentration to change?

Reply:
it only changes per module. In all likelihood it'll be the same for the other ones, originally I thought I might need very different values for the low affinity and high affinity experiments, but I think they're going to work well in the same range.